lib/src/postgrest_builder.dart

```diff
@@ -18,12 +20,30 @@ abstract class PostgrestBuilder {
   });
 
   dynamic body;
-  final List query = [];
   final Map<String, String> headers;
   bool maybeEmpty = false;
   String? method;
   final String? schema;
   Uri url;
+  PostgrestConverter? converter;
```

Comment:
I think this can be private. 

```suggestion
  PostgrestConverter? _converter;
```